en/docs/references/performance-analysis.md

```diff
@@ -1,18 +1,77 @@
+
+![Real-time performance analysis of the critical path](../assets/img/references/performance-analysis/real-time-performance-analysis.png){.cInlineImage-threeQuarter}
+
+By default, the Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click  **Reveal performance-critical path** on the top banner to highlight it in green in the low-code diagram.
+
+#### Latency estimates for individual API invocations in the performance-critical path
+
+The performance analyzer tool displays the estimated latency for each API invocation in the performance-critical path as a banner next to the API invocation in the low-code diagram, as shown below. The Performance Analyzer tool derives these estimates based on the user count range that you have selected.
+
+![Latency estimates for API Invocations in the performce-critical path](../assets/img/references/performance-analysis/latency-estimates-api-invocation.png){.cInlineImage-threeQuarter}
+
+### Analyze the performance of all execution paths
+
+By default, Performance Analyzer tool does not highlight the performance-critical path in the low-code diagram. Click on  **Reveal performance-critical path** on the top banner to highlight it in green. This opens a pane on the right side of the editor. In this pane, you can perform an in-depth performance analysis of the component. 
+
+![In-depth performance analysis](../assets/img/references/performance-analysis/in-depth-analysis.png){.cInlineImage-full}
+
+#### Execution paths
+
+You can view all execution paths in the component in a tabular format listed on top of the **Performance Forecast** pane. The table shows the entries in ascending order of the time it takes to process a single request in each path.
+
+![Execution paths](../assets/img/references/performance-analysis/execution-paths.png){.cInlineImage-half}
+
+The Performance Analyzer tool identifies the execution paths using the service invocations performed by each of them. If an execution path does not include any service invocation, then the latency of that path is assumed to be negligible.
+
+You can select any execution path from the table. When you select an execution path, the Performance Analyzer tool will highlight the execution path on the low-code diagram. The estimated overall performance values displayed on the top banner will change based on the execution path selected.
+
+#### Performance forecast graphs
+
+The performance forecast graphs, throughput, and latency, for the selected execution path, are found below the execution path table in the **Performance Forecast** pane.
```

Comment:
The performance forecast graphs for the selected execution path, are found below the execution path table in the **Performance Forecast** pane. These graphs show how the latency and throughput vary with the number of users accessing the component simultaneously.

Reply:
reworded as The performance forecast graphs for the selected execution path are below the execution path table in the **Performance Forecast** pane. These performance forecast graphs: the latency graph and the throughput graph, changes according to the number of users accessing the component simultaneously.